### PR TITLE
avoid duplicate textures on cache

### PR DESCRIPTION
**Pull request by ornitorrincos**
ugly solution for an ugly file.
(Believe it or not, both are needed)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -56,6 +56,10 @@ module.exports = function (PIXI)
         resource.data.spritesheets.forEach(function(spritesheet) {
             var name = spritesheet.image;
             var imageUrl = route + "/" + spritesheet.image;
+            PIXI.Texture.removeFromCache(name);
+            PIXI.Texture.removeFromCache(imageUrl);
+            PIXI.BaseTexture.removeFromCache(name);
+            PIXI.BaseTexture.removeFromCache(imageUrl);
```
Comment (uorbe001) [inline index.js]: why do you need to remove it from both the base texture and texture? aren't they the same?

Comment (ornitorrincos) [reply index.js]: surprisingly, not

Comment (grahamrhay) [reply index.js]: "lol"

Comment (uorbe001) [reply index.js]: for realz?

Comment (uorbe001) [reply index.js]: needs checking for those funcs as they don't exist on pixi 3

**Pull request merged**